src/format.rs

```diff
@@ -239,7 +239,7 @@ impl<'a> Formatter<'a> {
         }
     }
 
-    fn const_expr(&mut self, value: &ConstValue<()>) -> Document {
+    pub fn const_expr<T>(&mut self, value: &ConstValue<T>) -> Document {
```

Comment:
Does this need to be public and generic?

Reply:
I believe so, since `docs_const_expr` is used in docs and an error is thrown about associated functions since `docs_const_expr` uses `const_expr`. The value given from `docs_const_expr` would also be typed. I'll see if i can change it a little :)

Reply:
Ah yes! Makes sense. We can make it private though?

Reply:
Okay, at least i could remove pub is that good enough?